modules/dicom_archive/php/viewdetails.class.inc

```diff
@@ -304,7 +288,7 @@ class ViewDetails extends \NDB_Form
     /**
      * Overrides base getJSDependencies() to add additional JS files
      *
-     * @return array of extra JS files that this page depends on
+     * @return an array of extra JS files that this page depends on
```

Comment:
@llevitis 
It should return only [array] not [an array].
 @return array of extra JS files that this page depends on